projects/packages/backup/src/js/components/Admin/header.js

```diff
@@ -0,0 +1,58 @@
+import { JetpackVaultPressBackupLogo } from '@automattic/jetpack-components';
+import { useSelect } from '@wordpress/data';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import useCapabilities from '../../hooks/useCapabilities';
+import { STORE_ID } from '../../store';
+import { useIsFullyConnected } from './hooks';
+
+const Header = () => {
+	const isFullyConnected = useIsFullyConnected();
+
+	const { capabilities, capabilitiesError, capabilitiesLoaded, hasBackupPlan } = useCapabilities();
+	const noBackupCapabilities =
+		! hasBackupPlan &&
+		capabilitiesLoaded &&
+		! capabilitiesError &&
+		Array.isArray( capabilities ) &&
+		capabilities.length === 0;
+
+	const { activateLicenseUrl, shouldShowActivateLicenseLink } = useSelect(
+		select => {
+			const store = select( STORE_ID );
+			return {
+				activateLicenseUrl: `${
+					store.getSiteData().adminUrl
+				}admin.php?page=my-jetpack#/add-license`,
+
+				// This Boolean expression mirrors the logic found
+				// in the LoadedState component, in ./index.js
+				shouldShowActivateLicenseLink: isFullyConnected && noBackupCapabilities,
```

Comment:
Is there a reason to have this here when it's not tied to the store?

Reply:
You're 100% right about this. There's no reason for `shouldShowActivateLicenseLink` to be part of the `useSelect` expression, so I'll extract it. 👍